src/views/ReadArticle.js

```diff
@@ -305,8 +309,24 @@ export class Article extends Component {
                 <img className="share-icon" src={thumbsUp} alt="logo" />
                 <img className="share-icon" src={dislikeIcon} alt="logo" />
                 <img className="share-icon" src={bookmarkIcon} alt="logo" />
-                <img className="share-icon" src={facebookIcon} alt="logo" />
-                <img className="share-icon" src={twitterIcon} alt="logo" />
+                <a
+                  className="share-icon"
+                  href={`https://www.facebook.com/sharer/sharer.php?&u=${currentUrl}`}
+                >
+                  <img className="share-icon" src={facebookIcon} alt="logo" />
+                </a>
+                <a
+                  className="share-icon"
+                  href={`http://twitter.com/share?url=${currentUrl}&text=Hello world, also found this relevant&hashtags=authorsHeaven,software development`}
+                >
```

Comment:
@Musigwa can the text or tweet to share on twitter be related to an article? It may be the title of the article, the link of the article, or the portion of the article.

Reply:
Yes, I'll do that.